### ISSUE TITLE
Final level gave "perfect" when A/Z were damaged

### ISSUE THREAD

**Issue by IBwWG** [larsiusprime/tdrpg-bugs#740]
0.6.0.win7x64, HD mode@1704x960windowed
![dq-perfect-not](https://cloud.githubusercontent.com/assets/16719964/14975121/bd2973f4-10fe-11e6-9df7-0445c343e167.png)

**Comment by larsiusprime** [comment]
Okay, so I'm assuming you killed ET, and then you got a perfect++ which is wrong because both Azra & Zelemir have taken damage?

**Comment by IBwWG** [comment]
Correct.  I guess I could've been a lot clearer there re: whether ET died given the other issues lately :)
